### ISSUE TITLE
Creating a project inside the current directory doesn't work

### ISSUE THREAD

**Issue by dominikbraun** [verless/verless#175]
### Description

Creating a project inside the current directory using `verless create project .` or `verless create project` doesn't work:

```
RemoveAll .: invalid argument
```

### Reproduce

1. `verless create project --overwrite .`

### Expected behavior

A new project should be initialized.

### verless version

The output of `verless version`:

```
verless version v0.4.3
Git tag: v0.4.3
Git commit: b96616f
```

### System information

* OS: Windows 10
* Version: 1909

**Comment by hmble** [comment]
`os.RemoveAll` doesn't support path ending with `.` 
  [[1]](https://github.com/golang/go/blob/9706f510a5e2754595d716bd64be8375997311fb/src/os/path.go#L66) [[2]](https://github.com/golang/go/blob/9706f510a5e2754595d716bd64be8375997311fb/src/os/removeall_at.go#L22)

This repo uses `os.RemoveAll` with `--overwrite` option  [core/create.go#L38](https://github.com/verless/verless/blob/1b03f9503fe32439fe0ca62dabb8a83d869a046e/core/create.go#L38)

**Comment by dominikbraun** [comment]
> os.RemoveAll doesn't support path ending with .

Which makes sense. `os.RemoveAll` is used to ensure that the new project is a "fresh" directory, but this isn't necessary if it is created inside the current working directory.

A solution could be to only call `os.RemoveAll` if the path is _not_ the current working directory, e.g. using `if path != "."` or so.

**Comment by hmble** [comment]
I can make PR for this change.

**Comment by hmble** [comment]
Your suggested change creates project under current directory but doesn't remove existing content of directory with `--overwrite` command

```diff
-       if err := os.RemoveAll(path); err != nil {
-               return err
+       if path != "." {
+               if err := os.RemoveAll(path); err != nil {
+                       return err
+               }
```
If a user passes "." as path to --overwrite command, then should I remove all files in current directory and then create project files ?

**Comment by dominikbraun** [comment]
> If a user passes "." as path to --overwrite command, then should I remove all files in current directory and then create project files ?

Iif `--overwrite` is explicitily set, I think it would be ok to remove the contents of the current directory. However, if this is too extensive to implement for you, it would also be ok to just leave the files in the current directoy.

**Comment by hmble** [comment]
I managed to implement this functionality to remove contents of current directory with `--overwrite`. I will make PR
I will appreciate your feedback

**Comment by dominikbraun** [comment]
Fixed with #178.
